src/porepy/fracs/simplex.py

```diff
@@ -157,61 +157,68 @@ def triangle_grid(points, edges, domain, constraints=None, **kwargs):
     """
     logger.info("Create 2d mesh")
 
```

Comment:
This we should discuss. First, all the splitting functionality will be moved into FractureNetwork2d, so I am very hesitant to allow for changes now. 

Second, I have no clue what your changes here are doing; granted this was not a well documented function, but you sure didn't help out.